packages/maskbook/src/plugins/Trader/UI/trending/CoinMarketTable.tsx

```diff
@@ -92,10 +92,10 @@ export function CoinMarketTable(props: CoinMarketTableProps) {
                         {dataProvider !== DataProvider.UNISWAP_INFO ? (
                             <>
                                 <TableCell className={classes.cell} align="center">
-                                    {formatCurrency(trending.market?.circulating_supply ?? 0, '$')}
+                                    {formatToken(trending.market?.circulating_supply ?? 0)} {trending.coin.symbol}
                                 </TableCell>
                                 <TableCell className={classes.cell} align="center">
-                                    {formatCurrency(trending.market?.total_supply ?? 0, '$')}
+                                    {formatToken(trending.market?.total_supply ?? 0)} {trending.coin.symbol}
```

Comment:
@jack-works maybe can provide `FormattedToken` / `FormattedCurrency` components, simplify code.

Reply:
yeah

Reply:
i want's migrate `Wallet` folder to standalone sub-package, e.g: `@dimensiondev/wallet-utils`